base/array.jl

```diff
@@ -1791,13 +1791,16 @@ julia> findin(a,b) # 10 is the only common element
  4
 ```
 """
-function findin(a, b)
+function findin(a::Array{<:Real}, b::Union{Array{<:Real,Real}})
```

Comment:
I don't understand the union here. Did you mean `Union{Array{<:Real}, Real}`? Otherwise this is `Array{T, Real} where T<:Real`.